features/unprotected-temporary.json

```diff
@@ -22,6 +22,22 @@
         {
             "domain": "paramountplus.com",
             "reason": "https://github.com/duckduckgo/privacy-configuration/issues/1085"
+        },
```

Comment:
I would leave these off the temp list. Adding them to the temp allowlist will have the same affect as content blocking.